### PR TITLE
Get dependabot notifications only for top level dependencies

### PR DESCRIPTION
**Pull request by predragnikolic**

### CODE REVIEW

**Review by rchl — APPROVED** [review]

File: .dependabot/config.yml [hunk 1]
```diff
@@ -3,9 +3,4 @@ update_configs:
   - package_manager: "javascript"
     directory: "/server"
     update_schedule: "daily"
-    target_branch: "master"
-    allowed_updates:
-      - match:
-          dependency_type: "all"
-          update_type: "all"
-    version_requirement_updates: "increase_versions"
+    target_branch: "master"
```
Comment (rchl) [inline .dependabot/config.yml]:
```
	"ensure_newline_at_eof_on_save": true,
```
:)

Comment (rwols) [reply .dependabot/config.yml]: You guys don't have an LSP.sublime-project set up?


```
{
	"folders":
	[
		{
			"path": "."
		},
		{
			"path": "../LSP-eslint"
		},
		{
			"path": "../lsp_utils"
		},
		{
			"path": "../LSP-json"
		},
		{
			"path": "../UnitTesting"
		},
		{
			"path": "../LSP-vue"
		},
		{
			"path": "../LSP-css"
		},
		{
			"path": "../LSP-html"
		},
		{
			"path": "../LSP-intelephense"
		},
		{
			"path": "../LSP-typescript"
		},
		{
			"path": "../LSP-elm"
		},
		{
			"path": "../User"
		}
	],
	"settings":
	{
		"ensure_newline_at_eof_on_save": true,
		"rulers":
		[
			120
		],
		"tab_size": 4,
		"translate_tabs_to_spaces": true,
		"lsp_format_on_save": true,
		"LSP":
		{
			"pyls":
			{
				"command": ["pyls"],
				"scopes": ["source.python"],
				"syntaxes": ["Packages/Python/Python.sublime-syntax"],
				"languageId": "python",
				"env":
				{
					"PYTHONPATH": "/opt/sublime_text/Lib/python33",
				},
				"enabled": true,
			}
		}
	}
}
```

Comment (predragnikolic) [reply .dependabot/config.yml]: @rwols thanks :)

Although I am not sure how this part here works:
```
"LSP":
		{
			"pyls":
			{
				"command": ["pyls"],
				"scopes": ["source.python"],
				"syntaxes": ["Packages/Python/Python.sublime-syntax"],
				"languageId": "python",
				"env":
				{
					"PYTHONPATH": "/opt/sublime_text/Lib/python33",
				},
				"enabled": true,
			}
		}
```

I thought I needed a `clients` key between LSP and pyls:
```
"LSP": {
  "clients": {
    "pyls": {...}
```

Comment (predragnikolic) [reply .dependabot/config.yml]: BTW, that sublime project file could be added somewhere in the LSP docs, it is useful.

**Pull request merged**